setup.py

```diff
@@ -1,15 +1,15 @@
 import codecs
 import os
 
-import setuptools
+import setuptools  # type: ignore
```

Comment:
Rather than adding the ignores here, I'd just make the test command be `mypy mbq` so we ignore this file completely.